x-pack/solutions/observability/plugins/apm/public/components/shared/trace_waterfall/use_trace_waterfall.ts

```diff
@@ -125,7 +151,9 @@ export function getRootItemOrFallback(
 
   const parentIds = new Set(traceItems.map(({ id }) => id));
   // TODO: Reuse waterfall util methods where possible or if logic is the same
-  const orphans = traceItems.filter((item) => item.parentId && !parentIds.has(item.parentId));
+  const orphans = traceItems.filter(
+    (item) => item.parentId && !parentIds.has(item.parentId) && item.id !== rootItem?.id
```

Comment:
Can you add a comment here explaining why the new condition is now necessary? Maybe something like:
```suggestion
    (item) => 
    // Root cannot be an orphan.
    item.id !== rootItem?.id &&
    item.parentId &&
    !parentIds.has(item.parentId) 
```